### DISCUSSION TITLE
How do I tell if a patient or any of it's related resources has changed since a given date?

### DISCUSSION THREAD

**Discussion by johnkwaters** [FirelyTeam/firely-net-sdk#2086] [Q&A]
Say I fetched a patient and it's observations, medications, etc... with the everything call.
I store the timestamp for that call.
Is there a way to query the FHIR server for any changes to the patient or it's child resources that has happened since that timestamp?

**Comment by mbaltus** [comment]
Yes, there is, provided the server supports it. The definition of the everything operation (http://hl7.org/fhir/patient-operation-everything.html) specifies the _since input parameter, which does exactly what you want.

You will have to create a Parameters resource, add the parameter to it, and then use the FhirClient.InstanceOperation to execute the operation call.

**Reply by johnkwaters** [reply]
So, something like this?

```
        var since = new DateTimeOffset(
            new DateTime(2012, 02, 29, 12, 43, 0, DateTimeKind.Utc));
        var input = new Parameters
        {
            { "_count", new Integer(1000) },
            { "_since", new FhirDateTime(since) }
        };
        var resource = await fc.InstanceOperationAsync(
            ResourceIdentity.Build("Patient", patientId), "everything", input, useGet: true);

```

**Reply by mbaltus** [reply]
Yes, looks good!

**Reply by johnkwaters** [reply]
The server doesn't object to this query - but it returns the same data no matter what values I send - even if I send a future date.

**Reply by mbaltus** [reply]
Hmmm... could be that the parameters have not been implemented. Can you check the returned Bundle for any OperationOutcome resource, and if it is present, see what the message in it is?

**Reply by johnkwaters** [reply]
I checked, and no Outcome is present in the bundle.
